R/msqrob-utils.R

```diff
@@ -182,9 +175,37 @@ checkFullRank <- function(modelMatrix) {
     return(qr(modelMatrix)$rank == ncol(modelMatrix))
 }
 
+
 ##### None exported functions from multcomp package is included here to
 ##### During R and Bioc checks
 
+propagateFalseStatus <- function(vectors, statuses) {
```

Comment:
Nice attempt to solve the interaction issue, I however can't fully wrap my head around what you are doing. A few comments on the function: 
- Could you document the internal function using standard comments so to briefly describe what the function does, and especially what are the expected argument inputs.  
- Be very careful with repeat to not allow for infinite loops, I personally prefer a `while()` loop although there is still is risk of infinite loops.
- I can't see unit tests, but maybe this is in progress

Reply:
I added tests, commented the function and changed the code. https://github.com/statOmics/msqrob2/pull/64/commits/a97a4e5021bf49807b307cf9134156a32b261c0a